### PR TITLE
Smarter visited nodes

### PR DESCRIPTION
**Pull request by jacekkolasa**
solves
https://github.com/netdata/cloud-frontend/issues/3008

### CODE REVIEW

File: src/components/menus/item.js [hunk 1]
```diff
@@ -12,6 +12,18 @@ export const PanelRowContainer = styled(Flex)`
   ${props => props.selected && `background: ${getColor("selected")(props)};`}
 `
 
+const StyledIcon = styled(Icon)`
```
Comment (burbuli8ra) [inline src/components/menus/item.js:15]: I think `Icon` accepts `height` and `width` props, so maybe could be added as `attrs`! 😄

File: src/components/menus/item.js [hunk 2]
```diff
@@ -12,6 +12,18 @@ export const PanelRowContainer = styled(Flex)`
...
+  flex: 0 0 auto;
+  height: 16px;
+  width: 16px;
+`
+
+const StyledLoaderIcon = styled(IconComponents.LoaderIcon)`
```
Comment (burbuli8ra) [inline src/components/menus/item.js:21]: Same for this! 😄

**Review by burbuli8ra — APPROVED** [review]

File: src/components/menus/item.js (hunk 2 above)
Comment (novykh) [reply src/components/menus/item.js:21]: Does it?

**Review by novykh — APPROVED** [review]
Great, can we add the global css for scrollbars as well here, or different PR? [#2953](https://github.com/netdata/cloud-frontend/issues/2953)

File: src/components/menus/item.js (hunk 2 above)
Comment (burbuli8ra) [reply src/components/menus/item.js:21]: Do, you are right it doesn't!

**Comment by jacekkolasa** [comment]
@novykh sure, we can. But it's not that easy so i prefer separate issue/pr. Problem here is that the iFrame in Dashboard has some padding, so we cannot span the Panel for the whole width of sidebar :( We can change that in Dashboard, but it will create different look for users which won't upgrade their Agents

**Pull request merged**